modules/ROOT/pages/dataweave-system-properties.adoc

```diff
@@ -1,55 +1,31 @@
 = DataWeave System Properties
-
-
-DataWeave supports several system properties. To use these properties in application development and deployments, see xref:mule-runtime::mule-app-properties-system.adoc[System Properties].
-
+DataWeave supports several system properties. To use these properties in application development and deployments, see xref:mule-runtime:mule-app-properties-system.adoc[System Properties].
 [%header%autowidth.spread,cols=",a"]
 |===
 |Property |Description
-
```

Comment:
looks like com.mulesoft.dw.buffer.memory.monitoring was removed in 2.5?

Reply:
@andres-rad Was that property ever in 2.4? Should I also remove it from the 2.4 docs? https://docs.mulesoft.com/dataweave/2.4/dataweave-system-properties

Reply:
asked for details on channel

Reply:
Removed Feb 2021 in 2.4 per Andres

Reply:
TODO: update 2.4 docs about removal

Reply:
was never released in v2.4, so removing from v2.4 via https://github.com/mulesoft/docs-dataweave/pull/234